tests/kuttl/test-suites/default/watcher/04-assert.yaml

```diff
@@ -86,3 +94,25 @@ kind: Secret
 metadata:
   name: rabbitmq-transport-url-watcher-kuttl-watcher-transport
   namespace: watcher-kuttl-default
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rabbitmq-transport-url-watcher-kuttl-watcher-transport
+  namespace: watcher-kuttl-default
```

Comment:
same point as above about the namespace, we probably should not be hardcode it in the tests. Although since we already have instances were we do, it might be worth it to just remove all of them in a dedicated PR

Reply:
Yes, let's fix all of them for consistency in a dedicated PR